VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/confirmation/confirm-leave-hearing-popup.component.html

```diff
@@ -0,0 +1,31 @@
+<div id="confirmationDialog" class="vh-popup-overlay">
```

Comment:
Could the id be more specific? confirm-leave-hearing-modal or something. Camel case is also fine, I see that's used a lot of places like other modals. I don't think we have a specific format for ids.